lib/morphs/authenticable.js

```diff
@@ -46,26 +44,34 @@ module.exports = exports = function Authenticable(schema, options) {
     var authenticationFields = {};
 
     //prepare authentication field
-    authenticationFields[options.authenticationField] = {
-        type: options.authenticationFieldType,
-        unique: true,
-        //lowercase authentication field value before save
-        lowercase: true,
-        //trim authentication field
-        trim: true,
-        //add mongoose-valid8 validations
-        isRequired: {
-            message: options.authenticationField + ' is required'
-        }
-    };
+    if (options.authenticationField === 'email') {
+        authenticationFields[options.authenticationField] = {
+            type: String,
+            unique: true,
+            //lowercase authentication field value before save
+            lowercase: true,
+            //trim authentication field
+            trim: true,
+            validate: [validator.isEmail, 'Invalid email address {VALUE}'],
+            required: options.authenticationField + ' is required'
+        };
+    } else {
+        authenticationFields[options.authenticationField] = {
+            type: options.authenticationFieldType,
+            unique: true,
+            //lowercase authentication field value before save
+            lowercase: true,
+            //trim authentication field
+            trim: true,
+            validate: [validator.isEmail, 'Invalid email address'],
```

Comment:
This field can not be email address as it provided by implementer of irina. Consider removing it